frontend/public/components/_dropdown.scss

```diff
@@ -61,15 +61,9 @@ $dropdown-background--hover: $pf-color-black-200; // pf-c-dropdown__menu-item--h
 }
 
 .co-namespace-bar {
-  border-bottom: 1px solid $color-grey-background-border;
-  padding: 0 $pf-global-gutter;
+  border-bottom: var(--pf-global--BorderWidth--sm) solid var(--pf-global--BorderColor--100);
   white-space: nowrap;
 
-  @media (min-width: $pf-global--breakpoint--xl) {
-    padding-left: $pf-global-gutter--md;
-    padding-right: $pf-global-gutter--md;
-  }
-
```

Comment:
Not sure if this is the related change, but with this PR the padding on the right side of the dev perspective > topology view. See this toggle button to switch between graph and list view:

Before:

![image](https://user-images.githubusercontent.com/139310/157277648-aafd61c3-d0ee-4e0f-ac3b-4f4b11079d8c.png)

With this PR:

![image](https://user-images.githubusercontent.com/139310/157277703-8e0f94bb-b646-4012-897e-255e381a7850.png)

Reply:
In the admin page this looks unaligned also on the left side, but we have also created a bug on our side for this: https://bugzilla.redhat.com/show_bug.cgi?id=2042852

So I think it would be good to restore just the padding on the right side and we will follow up on the other aligments with this bug. Wdyt?

Before:

![image](https://user-images.githubusercontent.com/139310/157278600-30bd814f-ba47-4fb5-a038-c8a4e6a8f600.png)

With this PR:

![image](https://user-images.githubusercontent.com/139310/157278694-61feb29a-8203-49ba-b062-1d7d16c6ef13.png)

Reply:
Hey @sg00dwin, I'd hold off on updating to a `<PageSection>` for now, as they aren't intended to be nested. 

Instead, they're designed to stack in the DOM like this

![Screen Shot 2022-03-08 at 12 55 16 PM](https://user-images.githubusercontent.com/5385435/157297397-f91497eb-deaa-4c52-8827-6a1781eed9fd.png)

Being that the current structure of CO dashboard is this, I would expect `<PageSection>`s to behave unpredictably (especially if nested):

![Screen Shot 2022-03-08 at 12 58 24 PM](https://user-images.githubusercontent.com/5385435/157297687-e303aebe-e7c4-400b-a422-ffb8986d09a8.png)

I'd love to revisit updating to a structure more aligned with PF, if our timeline allows, to something more like this:

![Screen Shot 2022-03-08 at 1 06 37 PM](https://user-images.githubusercontent.com/5385435/157298984-4a9fc6a4-673a-4072-98b2-99b70bd6bacc.png)

Reply:
> In the admin page this looks unaligned also on the left side, but we have also created a bug on our side for this: https://bugzilla.redhat.com/show_bug.cgi?id=2042852
> 
> So I think it would be good to restore just the padding on the right side and we will follow up on the other aligments with this bug. Wdyt?
> 

Ok, I've reverted earlier changes with right side padding changes.

Reply:
Thanks. Looks good again :+1: